src/sentry/models/integrations/sentry_app.py

```diff
@@ -119,7 +120,7 @@ class SentryApp(ParanoidModel, HasApiScopes, Model):
     owner_id = HybridCloudForeignKey("sentry.Organization", on_delete="CASCADE")
 
     name = models.TextField()
-    slug = models.CharField(max_length=SENTRY_APP_SLUG_MAX_LENGTH, unique=True)
+    slug = SentrySlugField(max_length=SENTRY_APP_SLUG_MAX_LENGTH, unique=True, db_index=False)
```

Comment:
Should this include `db_index=False`?

Reply:
Yup! `CharField` does not index normally, and `SlugField` does index by default. Thus to keep any migrations out, we need to align the defaults.